ercc/ercc.go

```diff
@@ -156,6 +157,22 @@ func (ercc *EnclaveRegistryCC) registerEnclave(stub shim.ChaincodeStubInterface,
 	// set enclave public key in attestation report
 	attestationReport.EnclavePk = enclavePkAsBytes
 
+	// second, check that attestation report contains expectedMrEnclave as defined in chaincode definition
+	// TODO set chaincode name
+	expectedMrEnclave, err := utils.ExtractMrEnclaveFromChaincodeDefinition("chaincode  name", stub)
```

Comment:
I expect this to fail. however, on my local tests it doesn't. I hope travis catches this.

Reply:
well, travis also succeeded. So i guess something must be wrong ...

Reply:
The tests did not fail because `ercc.ra.CheckMrEnclave(expectedMrEnclave, attestationReport)` always returns true in simulation mode. :) so whatever is read from the chaincode definition (or not) is just ignored by the actual check. Anyway, I commented out all the useful checks in ercc but keep it there to help Bruno to integrate this during ercc refactoring.